sdk/schemaregistry/schema-registry-avro/test/schemaRegistryAvroSerializer.spec.ts

```diff
@@ -45,7 +45,7 @@ const testGroup = "azsdk_js_test_group";
 const testSchemaIds = [
   "{773E17BE-793E-40B0-98F1-0A6EA3C11895}",
   "{DC7EF290-CDB1-4245-8EE8-3DD52965866E}"
-].map((x) => x.replace(/[\{\-\}]/g, ""));
+].map((x) => x.replace(/[{\-}]/g, ""));
```

Comment:
@abc516 Can you share which linting rule failure is this change fixing?

@nguerrera, Do you recall what exactly are we trying to replace here and why?

Reply:
@ramya-rao-a @nguerrera this fixes this rule: https://eslint.org/docs/rules/no-useless-escape .

I believe it just subs out the '{', '-', and '}' chars in the guid strings with blank spaces. Note the test below from https://regexr.com , using the new expression :
![image](https://user-images.githubusercontent.com/8355075/97175829-7c0b0000-176a-11eb-9faf-34bcf1ebfd93.png)

Reply:
Yes, that's what it does and this is fine. :smile:

Reply:
Thanks Nick